src/misc/like.ts

```diff
@@ -0,0 +1,241 @@
+}
+
+/**
+ * Calculates milliseconds to hours and minutes
+ */
+function calculateTime(duration: number) {
+  let minutes = Math.floor((duration / (1000 * 60)) % 60),
+    hours = Math.floor((duration / (1000 * 60 * 60)) % 24);
+  hours = hours < 10 ? 0 + hours : hours;
+  minutes = minutes < 10 ? 0 + minutes : minutes;
+
+  return { hours, minutes };
+}
+
+/**
+ * Display player record of the server
+ */
+registerCommand(
+  ['suosikit', 'favourites'],
+  (sender) => {
+    sender.sendMessage('§6--------------------------------------');
+    sender.sendMessage('§eValtakauden suosituimmat pelaajat:');
+    sender.sendMessage(' ');
+    let index = 0;
+    for (const user of getLiked()) {
+      if (!user) continue;
+      index++;
+      if (
+        hasTimePassedToDecrease(
+          getLikeRemovalTimestamp(
+            Bukkit.getOfflinePlayer(UUID.fromString(user)),
+          ),
+        )
+      ) {
+        removeLike(user, () => {
+          setLikeRemovalTimestamp(
+            Bukkit.getOfflinePlayer(UUID.fromString(user)),
+          );
+          return;
+        });
+      }
+      const name =
+        Bukkit.getOfflinePlayer(UUID.fromString(user))?.name ||
+        'tuntematon pelaaja';
+      sender.sendMessage(
+        `§eSijalla §6#${index} §eon ${name}, ${getLikes(
+          Bukkit.getOfflinePlayer(UUID.fromString(user)),
+        )} tykkäystä`,
+      );
+    }
+    sender.sendMessage('§6--------------------------------------');
+  },
+  {
+    permission: 'vk.player-record',
+    description: 'Valtakauden pelaajaennätys',
+  },
+);
+
+registerCommand(
+  ['tykkää', 'tykkaa', 'like'],
+  (sender, _label, args) => {
+    if (args.length == 0) {
+      sender.sendMessage('§6--------------------------------------');
+      sender.sendMessage(
+        '§eTykkää muista pelaajista komennolla §6/tykkää <nimi>§e!',
+      );
+      sender.sendMessage('§6--------------------------------------');
+    } else {
+      if (!(sender instanceof Player)) return;
+      if (Date.now() < getCooldownTimestamp(sender as Player)) {
+        const time = calculateTime(
+          getCooldownTimestamp(sender as Player) - Date.now(),
+        );
+        sender.sendMessage(
+          `§6Voit tykätä uudelleen ${time.hours} tunnin ja ${time.minutes} minuutin kuluttua`,
+        );
+        return;
+      }
+      if (args[0].toLowerCase() == sender.name.toLowerCase()) {
+        sender.sendMessage('§6Tiedämme jo että pidät itsestäsi!');
+      } else if (isAdminAccount(args[0].toLowerCase())) {
+        sender.sendMessage('§6Et voi tykätä ylläpitotileistä!');
+      } else {
+        const player = Bukkit.getOfflinePlayer(args[0]);
+        if (!player || !player.uniqueId) {
+          sender.sendMessage('§6Pelaajaa ei löydetty.');
+          return;
+        }
+        addLike(player.uniqueId.toString(), () => {
+          sender.sendMessage(`§6Tykkäsit pelaajasta §e${player.name}§6!`);
+          setCooldownTimestamp(sender as Player);
```

Comment:
Could this callback be inside the addLike function? How about passing the sender as argument?
Maybe something like:
addLike(from, to)